devices/microsoft/research/jmrotarycontrolbuttonv11.json

```diff
@@ -6,7 +6,9 @@
     "version": "1.1",
     "repo": "https://github.com/microsoft/jacdac-msr-modules",
     "link": "https://github.com/microsoft/jacdac-msr-modules",
+    "hardwareDesign": "https://github.com/microsoft/jacdac-ddk/tree/main/electronics/altium/reference-designs/JacdacRotary%2026-1.2",
```

Comment:
Could we remove the space from these files to avoid having escaped space in URLs? (no more %20 please)

Reply:
that will require a script as there are spaces everywhere  under the altium directory, and I am not sure that it wouldn't break altium unless we go into files and make sure any references to the files are renamed. Please file a separate issue.